docker-compose-db.yml

```diff
@@ -26,6 +26,24 @@ services:
     env_file:
       .env-db
 
+  nginx:
```

Comment:
db_nginx would be a better name otherwise one would think that it's the production server of the website

Reply:
Is this necessary? I think that the nginx server could be used for both (DB and frontend).
The root of the webserver (https://lab.idiap.ch/enermaps) could redirect for example to the frontend (`localhost:7000`), if we keep together all the services in the same `docker-compose.yml` as it was at the beginning

Reply:
Actually yes it is necessary.
For multiple reasons:
1) the services could run on different servers: for example the db is currently on nginx while the website could use Apache
2) if the website is handled by someone else after the deployment, this could be a security issue if no access restrictions are present for the different services
3) it's a requirement to separate both services
4) it would become difficult for the IT team to calculate the effective cost and resources usage for each service if no proper separation is in place (specially taking into account that the website is managed by a different entity which is the CREM here)
5) at the moment for the deployment of the website no decision is taken with respect to what WSGI and server will be used, what are the security requirements, etc.